index.html

```diff
@@ -113,11 +114,13 @@ <h4>About Me</h4>
      <section id="contact" class="get-intouch">
          <h4>I'm always intrested in hearing about new projects, so 
              if you'd like to get chat please get in touch</h4>
-             <form action="https://formspree.io/f/moqrdrwj" method="POST">
-                 <input type="text" name="name" maxlength="30" placeholder="Full name" required>
-                 <input type="email" name="email" placeholder="Email address" required>
+             <form id="form" action="https://formspree.io/f/moqrdrwj" method="POST">
+                 <input type="text" id="username" name="name" maxlength="30" placeholder="Full name" required>
+                 <input type="email" id="email" name="email" placeholder="Email address" >
                  <textarea name="Message" maxlength="500" placeholder="Enter text here" required></textarea>
+                 <small></small>
                  <button type="submit">Get in touch</button>
+                 
              </form>
```

Comment:
- [ ] Great job @sainingo! On the mobile version, all validations are working. But, I can't submit the form after filling it out. On the desktop version, any validations are not working. Kindly fix the issue so on the mobile and desktop versions your contact form works as expected.